src/classes/classes-public.ts

```diff
@@ -0,0 +1,83 @@
+//public classes
+
+/*
+
+Classes are a common abstraction used in object-oriented programming (OOP) languages to describe data structures known as objects.​
+
+You can think of the class itself as a blueprint for creating objects with the given shape, while instances are the objects themselves, created from this blueprint.​
+
+​
+
+You can create a class declaration by using the class keyword, followed by the class name and then a {} pair block.​
+
+You can then create a new instance of the Person class by using the new keyword followed by the name of your class.​
+
+
+
+Class members in TypeScript may have three possible visibility modifiers: public, protected, and private. 
+public members may be accessed outside of the class instance, where as private ones cannot. 
+protected occupies a middle ground between the two, where members can be accessed by instances of the class or subclasses based on that class.​
+*/
+class Person {
+  constructor(public name: string, public age: number) {}
```

Comment:
From the viewpoint of a person who sees classes for the first time, I believe it would be better if we also explain what a constructor is.